.golangci.yml

```diff
@@ -29,7 +29,7 @@ linters-settings:
 
   godot:
     capital: true
-    period: false
+    period: true
```

Comment:
Not sure if your branch is based on main, but this value is already set to rue in `main`.

https://github.com/percona/pmm/blob/6e8cc78054950739598adb182ef16d6f851f35e5/.golangci.yml#L32

Reply:
Moreover, you previous PR was merged, so there is no need to put the dots at the end of comments. Please double check to make sure your branch is based on the latest `main`.